docker-compose.yml

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
+#
+# This file is part of perseo-fe
+#
+# perseo-fe is free software: you can redistribute it and/or modify it under the terms of the GNU Affero
+# General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your
+# option) any later version.
+# perseo-fe is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the
+# implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+# for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along with perseo-fe. If not, see
+# http://www.gnu.org/licenses/.
+#
+# For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
+#
+
+version: '3.4'
+services:
+    perseo-mongo:
+        image: mongo:3.6
```

Comment:
Seems to be pretty obsolete... Should be the same used in other places in this repo (.github/workflows/ci.yml uses 4.2, for instance)

Reply:
Fixed in https://github.com/telefonicaid/perseo-fe/pull/592/commits/fd872d8922203ee91da0747b762e37240c3a7973